### DISCUSSION TITLE
有生之年！！！《凉宫春日的剧场》简体中文版竟然同步发布！！！

### DISCUSSION THREAD

**Discussion by Uknown-wang** [share121/inter-knot#4494] [灌水]
引用来自天闻角川的消息，凉宫春日系列终于出续集了，还是全球同步发售（这次没有环大陆发布了）
BYD的谷川流终于把麻将搓完了！！！
![640](https://github.com/user-attachments/assets/38dd71b7-5b9a-4c2a-a119-04da56db4d87)

**Comment by EvilPetal** [comment]
有生之年！

**Comment by my-monkey145** [comment]
爽啦！！！<img src="https://bbs-static.miyoushe.com/static/2024/07/31/cd2ca0d9143e10e588e0a0191bf71c3b_5335236511308035335.png" alt="素裳 喷火" width="75" >

**Comment by dinkaka** [comment]
到时候直接去线下买
